phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -128,6 +128,8 @@ abstract class Package_PHP_XHTML extends Package_Generic_XHTML {
     /* Current Chunk settings */
     protected $cchunk          = array();
     /* Default Chunk settings */
```

Comment:
This comment is now in the wrong place. It belongs on the `$dchunk` property.

Reply:
Oops.